pkg/api/mlflow/service/artifact/storage/storage.go

```diff
@@ -31,13 +32,13 @@ func (o ArtifactObject) IsDirectory() bool {
 	return o.IsDir
 }
 
-// ArtifactStorageProvider provides and interface to work with particular artifact storage.
+// ArtifactStorageProvider provides an interface to work with artifact storage.
 type ArtifactStorageProvider interface {
-	// List lists all artifact object under provided path.
 	List(artifactURI, path string) ([]ArtifactObject, error)
+	GetArtifact(artifactURI, path string) (io.ReadCloser, error)
```

Comment:
can you please get back comments and add comment for new method?

Reply:
also, can you please rename `GetArtifact` to just `Get`?

Reply:
yes can do